docs/graphcast/radios/one-shot.md

```diff
@@ -0,0 +1,35 @@
+| `GRAPH_ACCOUNT`     | Graph account corresponding to Graphcast operator.<br/>Example: `GRAPH_ACCOUNT=YOUR_GRAPH_ACCOUNT`                                                                                                         |
+| `REGISTRY_SUBGRAPH` | Subgraph endpoint to the Graphcast Registry.<br/>Default: `https://api.thegraph.com/subgraphs/name/hopeyen/graphcast-registry-goerli`                                                                      |
+| `NETWORK_SUBGRAPH`  | Subgraph endpoint to The Graph network subgraph.<br/>Default: `https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-goerli`                                                                 |
+| `GRAPHCAST_NETWORK` | Supported Graphcast networks: mainnet, testnet.<br/>Default: `testnet`                                                                                                                                     |
+| `TOPICS`            | Comma separated static list of content topics to subscribe to.<br/>Example: `TOPICS=topic1,topic2,topic3`                                                                                                  |
+| `IDENTIFIER`        | Subgraph hash is used to be the message content identifier.<br/>Example: `IDENTIFIER=YOUR_IDENTIFIER`                                                                                                      |
+| `NEW_HASH`          | Subgraph hash for the upgrade version of the subgraph.<br/>Example: `NEW_HASH=YOUR_NEW_HASH`                                                                                                               |
+| `SUBGRAPH_ID`       | Subgraph id shared by the old and new deployment.<br/>Example: `SUBGRAPH_ID=YOUR_SUBGRAPH_ID`                                                                                                              |
+| `INDEX_NETWORK`     | Subgraph id shared by the old and new deployment.<br/>Example: `INDEX_NETWORK=YOUR_INDEX_NETWORK`                                                                                                          |
+| `MIGRATION_TIME`    | UNIX timestamp that the developer plan on migrating the usage.<br/>Example: `MIGRATION_TIME=YOUR_MIGRATION_TIME`                                                                                           |
+| `ID_VALIDATION`     | Identity validation mechanism for message signers (options: no-check, valid-address, graphcast-registered, graph-network-account, registered-indexer, indexer).<br/>Example: `ID_VALIDATION=valid-address` |
+| `LOG_LEVEL`         | Logging configuration to set as RUST_LOG.<br/>Default: `info`                                                                                                                                              |
+| `SERVER_HOST`       | If port is set, the Radio will expose API service on the given host.<br/>Default: `0.0.0.0`                                                                                                                |
```

Comment:
one-shot doesn't have host a server, so this doesn't affect anything; this is my fault from leaving it in the code. perhaps delete here in the meantime
`SERVER_PORT` and `TOPICS` too